src/sentry/incidents/endpoints/organization_alert_rule_available_action_index.py

```diff
@@ -11,20 +11,43 @@
 from sentry.incidents.endpoints.serializers import action_target_type_to_string
 from sentry.incidents.logic import get_available_action_integrations_for_org
 from sentry.incidents.models import AlertRuleTriggerAction
+from sentry.models import OrganizationIntegration, PagerDutyService
 
 
 class OrganizationAlertRuleAvailableActionIndexEndpoint(OrganizationEndpoint):
-    def build_action_response(self, registered_type, integration=None):
-        return {
+    def fetch_services(self, organization, integration_id):
```

Comment:
@ceorourke I'd name this something like `fetch_pagerduty_services` so it's more clear